packages/control-property-editor/src/panels/RightPanel.tsx

```diff
@@ -42,6 +42,7 @@ export function RightPanel(): ReactElement {
                 scrollable={true}
                 layout={UISectionLayout.Standard}
                 className="editor__quickactions"
+                bodyClassName="app-panel-scroller"
```

Comment:
There are additional styles also coming with this class and it breaks spacing 
![image](https://github.com/user-attachments/assets/42156841-44b0-4719-8468-f46afe476d42)

Having to set scrollbar styles for every single element seems error prone. For example, styles for properties panel are missing. 
![image](https://github.com/user-attachments/assets/a53566aa-0bf3-4560-a51e-37b3ea1620f5)

It would be better to set them globally. The drawback is that the we can't add hover effects, but the web views in vscode function the same way.